lib/manager/pre-commit/readme.md

```diff
@@ -1,4 +1,10 @@
-Renovate supports updating of Git dependencies within pre-commit configuration `.pre-commit-config.yaml` files or other YAML files that use the same format (via `fileMatch` configuration).
+_Important note_: The `pre-commit` manager is disabled by default and must be opted into through config.
+Renovate's approach to version updating is not fully aligned with `pre-commit upgrade` and this has caused frustration for `pre-commit`'s creator/maintainer.
+Attempts to work with the `pre-commit` project to fix these gaps have been rejected, so therefore we have chosen to disable the manager by default indefinitely.
+Please do not contact the `pre-commit` project/maintainer about any Renovate-related topic as the result is unlikely to be productive.
```

Comment:
```suggestion
Please do not contact the `pre-commit` project/maintainer about any Renovate-related topic.
```

I don't think we're hitting the right tone of voice here. It feels a bit snarky to say: "Don't bother contacting them as the result is unlikely to be productive."

I think we're better off just sticking to our request: "Do not contact maintainer when you have problems with Renovate-related topics".